_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -23606,11 +23606,10 @@
 /area/science/lab)
 "bhE" = (
 /obj/structure/table,
-/obj/item/stack/sheet/glass/fifty{
-	pixel_x = 3;
-	pixel_y = 3
+/obj/item/stack/sheet/metal/fifty{
+	amount = 5;
```

Comment:
what

Just use /obj/item/stack/sheet/metal instead of vareditting /obj/item/stack/sheet/metal/fifty from fifty to five.

Reply:
done, pushing now

Reply:
pushed